cgp/individual.py

```diff
@@ -105,7 +106,9 @@ def __repr__(self):
         return f"Individual(idx={self.idx}, fitness={self.fitness}, genome={self.genome}))"
 
     def clone(self) -> "IndividualSingleGenome":
-        return IndividualSingleGenome(self.fitness, self.genome.clone())
+        ind = IndividualSingleGenome(self.fitness, self.genome.clone())
+        ind.parent_idx = self.idx
```

Comment:
This is a bit tricky, but I don't think that we can generally say that the clone of an individual has the individual as its parent, can we?

Reply:
uhm, i'm not sure. what other definition of "parent" is there other than the one from which we create new individuals? maybe i'm misunderstanding your questions :thinking:

Reply:
Yes, okay, I think this makes sense.